src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java

```diff
@@ -20,4 +31,39 @@ public KeepersSlackCommandController(KeeperService keeperService, SlackNameHandl
         this.keeperService = keeperService;
         this.slackNameHandlerService = slackNameHandlerService;
     }
+
+    @PostMapping(value = "/commands/keeper", consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE)
+    public RichMessage onReceiveSlashCommandGetKeeperDirections(@RequestParam("token") String token,
+                                                                @RequestParam("user_name") String fromUser,
+                                                                @RequestParam("text") String text) {
+        logger.debug("Received slash command GetKeeperDirections: from user: [{}] command: [{}] token: [{}]",
+                fromUser, text, token);
+
+        if (!token.equals(slackToken)) {
+            logger.warn("Received invalid slack token: [{}] in command Keeper for user: [{}]", token, fromUser);
+            return new RichMessage("Sorry! You're not lucky enough to use our slack command.");
+        }
+        logger.debug("Started create slackParsedCommand");
+        SlackParsedCommand slackParsedCommand = slackNameHandlerService.createSlackParsedCommand(fromUser, text);
+        logger.debug("Finished create slackParsedCommand: " + slackParsedCommand.toString());
+
+        String keeperUuid = slackParsedCommand.getFirstUser().getUuid();
+        String keeperSlackName = slackParsedCommand.getFirstUser().getSlack();
+
+        List<String> result = keeperService.getKeeperDirections(keeperUuid);
+        logger.debug("Received response from Keeper service: [{}]", result.toString());
+
+        String response = "ERROR. Something went wrong and we didn't get keeper directions";
+        if (result.size() == 0) {
+            response = "За Хранителем " + keeperSlackName + " сейчас не закреплено ни одно направление.";
```

Comment:
In specification those lines stated in Russian (SLB-F3-RSP). If so, I change them.